src/components/HelpfulLinksBox.js

```diff
@@ -9,15 +9,15 @@ import * as Utils from 'src/libs/utils'
 const HelpfulLinksBox = ({ method }) => {
   return div({ style: { backgroundColor: colors.accent(0.2), paddingTop: '0.25em', paddingBottom: '0.25em', paddingLeft: '1em', paddingRight: '1em' } }, [
     h4('Have questions?'),
-    isCovid19Method(method.name) && p([
+    isCovid19Method(method?.name) ? p([
```

Comment:
I think`?` can be reverted to `&&` and it should still work as expected. This way `: null` on L19 won't be necessary